src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs

```diff
@@ -663,7 +663,14 @@ public string LiteralPath
             get => _path;
             set
             {
-                _path = value;
+                var paths = SessionState.Path.GetResolvedProviderPathFromPSPath(value, out _);
```

Comment:
It seems to me that this operation should be done in the `ProcessRecord` method instead.

Reply:
We could? But I considered that if the input is invalid or resolves to multiple paths then there's no way for it to reach ProcessRecord at all, since the backing field here is `string` and not `string[]`.